ats/atsMachines/fluxScheduled.py

```diff
@@ -0,0 +1,115 @@
+#ATS:slurm56 SELF FluxScheduled 3
+#ATS:toss_4_x86_64_ib SELF FluxScheduled 16
+
+"""
+This module defines the ``FluxScheduled`` class.
+"""
+
+import time
+from math import ceil
+
+import flux
+import flux.job
+
+from ats import terminal
+from ats.tests import AtsTest
+from ats.atsMachines import lcMachines
+
+## Do we need to request an allocation? On fluke if you issue Flux mini run it sends to a compute node automatically
+## but there's no assurances it will be to the _same_ compute nodes for each of ats's "tests" or whatever
+## answer: YES for now ## Should start its own Flux session!! TODO
+
+FLUX_HANDLE = flux.Flux()  ## where do I go?
```

Comment:
I would say it should be an instance attribute. It's fine as a module constant like it is, but I don't really see why it should be, and I generally think that if there isn't a reason for it to be a module-level constant or a class attribute, it should be an instance attribute.

I would say in general:

- Module constant/attribute if it needs to be shared among functions or classes
- Class attribute if it needs to be shared among instances
- Otherwise, it should be an instance attribute

Instance attributes generally go in the `__init__` method, e.g.

```python

def __init__(self, *args, **kwargs):
    self._flux_handle = flux.Flux()
    super().__init__(*args, **kwargs)
```

Although it looks like this class has an `init` method which also defines instance attributes. That's not standard but there may be good reasons for it.